docs/Modding/Tilesets/graphics.rst

```diff
@@ -0,0 +1,446 @@
+  used.
+
+  Currently the citizen and specialist sprites may not have any transparency, as this is ignored in much of
+  the drawing. This is considered a bug.
+
+Specialist Sprites
+  These provide specialist graphics just like the citizen graphics. However, specialist types come from the
+  ruleset and may be changed in modpacks. The sprite name is "specialist.<type>_<n>". Again <type> is the
+  type of specialist (currently "elvis", "scientist", "taxman") while <n> is the sprite number. See "citizen
+  sprites" above.
+
+Progress Indicators
+  There are three types of progress indicator. "science_bulb" indicates progress toward the current research
+  target. "warming_sun" indicates progress toward global warming. "cooling_flake" indicates progress toward
+  nuclear winter. Each indicator should have 8 states, numbered 0 (least) through 7 (most). The sprite names
+  are "s.<type>_<n>".
+
+Government Icons
+  There should be one icon for each government. Its name is "gov.<gov>", where <gov> is the government name.
+  Government types come from governments.ruleset (currently "anarchy", "despotism", "monarchy", "communism",
+  "fundamentalism", "republic", "democracy").
+
+Tax Icons
+  One icon for each tax type. These are used to show the tax rates. The sprites are "s.tax_luxury",
+  "s.tax_science", "s.tax_gold". Commonly the specialist sprites are reused for this.
+
+Right Arrow
+  A sprite "s.right_arrow" is used on the panel when more units are present than can be shown.
+
+
+Terrain Sprites
+---------------
+
+Base Sprite
+  If the terrain has no match type or is layered, a base sprite is needed. This sprite has tag "t.<terrain>1"
+  (e.g., "t.grassland1"). More than one such sprite may be given ("t.grassland2", etc.) in which case one will
+  be chosen at random for each tile.
+
+Matched Sprites
+  If the terrain has a match type or is layered, a set of matched sprites is needed. This consists of 16
+  sprites with tags "t.<terrain>_n<V>e<V>s<V>w<V>" (e.g., "t.hills_n0e0s1w0". Each direcional value <V> is
+  either 0 or 1. Note that the directions are in map coordinates, so n (north) in iso-view is northeast on the
+  mapview. (Note this only applies for cell_type "single".)
+
+Cell Sprites
+  For matched terrains that have cell_type "rect", 32 different sprites are needed. Each sprite is a rectangle
+  corresponding to one cell, and there are 8 different sprites per cell. Each sprite has a name like
+  "t.ocean_cell_u110" where "ocean" is the terrain, "u" means up (north on the map) and 110 indicates which of
+  the adjacent tiles are mismatched. For instance u110 means:
+
+.. code-block:: rst
+
+      |      /\
+      |     /B \
+      |    /\ 1/\
+      |   / A\/C \
+      |   \1 /\ 0/
+      |    \/D \/
+      |     \  /
+      |      \/
+
+
+a matching terrain exists at C but not at A or B. In this case D is the current tile.
+
+  Examples:
+
+.. code-block:: rst
```

Comment:
Maybe `ini` will result in better syntax highlighting

Reply:
it does